README.md

```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+
+## Launch The Bot Using Docker
+Prerequisites
+
+Make sure you have already installed both Docker Engine and Docker Compose. You don’t need to install other programs like Python  as they are provided by Docker images.
+
+1. You can see how to start with docker compose here.
+2. To run the bot locally  with Docker, we need to set Passing Environment Variables Into the Dockerfile you created in 1.
+3. To do that, When we want to set this argument, we pass it with the –build-arg flag:
+
+docker build -t build-name --build-arg name=arg-name.
+4. Read about it in detail here  
```

Comment:
There are no details to be read here :trollface: we can remove this point

Reply:
yeah, i thought I had inserted a hyperlink but it didn't appear, unfortunately :(